### ISSUE TITLE
Fine-tuned model on monkey anatomy?

### ISSUE THREAD

**Issue by alexisthual** [BBillot/SynthSeg#53]
Hi!
Thank you for the very nice package :blush: 
I don't know if this is the right place, but I was interested to know if anyone had tried the pre-trained models on monkey images, and if fine-tuning them on such data would yield interesting results.

**Comment by BBillot** [comment]
Hi, and thanks for the interest ! :)

I don't think anyone has tried SynthSeg on monkey brains, but let me know the results if you do ! But regardless, I think that fine-tuning would definitely improve the results. 

Ideally, you would fine-tune by using the same domain-randomisation technique as for SynthSeg, but I guess you can also fine-tune by using normal supervised data (ie real images with corresponding segmentations). In both cases, you would have to be careful about the training label maps, because if you want the fine-tuned model to segment the same labels as SynthSeg, then these training segmentations would have to contain all these labels. But I'll leave it up to you ;) 

Let me know how it goes,
Best
Benjamin

**Comment by alexisthual** [comment]
Good morning!
For future reference, I have tried segmenting a macaque T1 image, and it did not turn-out well haha.
![Screenshot from 2023-05-11 12-00-41](https://github.com/BBillot/SynthSeg/assets/13835654/f7513f4c-1ad6-403e-b3c4-1b9196f37986)

I might try to fine-tune synthseg 2.0 some day with labels and domain-randomisation techniques that match that of the current training procedure, but it's not on my todo list at the moment :)

**Comment by BBillot** [comment]
Sorry but are you sure the affine matrix in the header correctly represents the image ? Because having a wrongly described orientation would be a problem for SynthSeg and I see here that the image is displayed in a weird orientation.

**Comment by BBillot** [comment]
Any news on this ? 
As I said, I strongly suspect the image header to be corrupted. SynthSeg needs the header to re-align the input images to RAS coordinates, but if the header doesn't have the correct orientation, then the image won't be aligned to RAS and SynthSeg will give you a bad segmentation.
Of course, there is always the possibility of SynthSeg not working on monkey brains, but I think we need to check the header first.
Sorry to trouble you, it's just that I'm very curious about this ! :)

Best,
Benjamin

**Comment by alexisthual** [comment]
Sorry I did not get back to you earlier!
I did not try anything new on this and probably won't have time to dedicate to that, but I'm very interested if someone wants to jump in!
